src/actions/debug/debug.ts

```diff
@@ -3,9 +3,7 @@ import * as winston from "winston"
 import * as Hub from "../../hub"
 
 export function registerDebugAction() {
-  if (process.env.ACTION_HUB_DEBUG_ENDPOINT) {
-    Hub.addAction(new DebugAction())
-  }
+  Hub.addAction(new DebugAction())
```

Comment:
I might have missed this in a previous commit - but do we want to update the Debug action here?  It's unrelated to the commit topic so I'm inclined to do one more commit to revert the debug changes.